src/main/java/graphql/execution/SubscriptionExecutionStrategy.java

```diff
@@ -103,13 +103,18 @@ public SubscriptionExecutionStrategy(DataFetcherExceptionHandler dataFetcherExce
     }
 
     private ExecutionResult wrapWithRootFieldName(ExecutionStrategyParameters parameters, ExecutionResult executionResult) {
-        String rootFieldName = parameters.field().get(0).getName();
+        String rootFieldName = getRootFieldName(parameters);
         return new ExecutionResultImpl(
                 singletonMap(rootFieldName, executionResult.getData()),
                 executionResult.getErrors()
         );
     }
 
+    private String getRootFieldName(ExecutionStrategyParameters parameters) {
+        Field rootField = parameters.field().get(0);
+        return rootField.getAlias() != null ? rootField.getAlias() : rootField.getName();
```

Comment:
@bbakerman do you think a null check is enough, or should we check if the string is empty too?

Reply:
Its probably ok but I would do both.  Pretty sure null if the alias is missing but beltcs and braces